installation/scripts/append-configmap-from-versions-env-to-installer-yaml.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+KYMA_PATH="${CURRENT_DIR}/../.."
+VERSIONS_ENV_PATH="${KYMA_PATH}/installation/versions.env"
+INSTALLER_YAML_PATH="${KYMA_PATH}/installation/resources/installer.yaml"
+
+if [ ! -f ${VERSIONS_ENV_PATH} ]; then
+    echo "${VERSIONS_ENV_PATH} not found"
+    exit 1
+fi
+
+if [ ! -f ${INSTALLER_YAML_PATH} ]; then
+    echo "${INSTALLER_YAML_PATH} not found"
+    exit 1
+fi
+
+TMPFILE=$(mktemp ${KYMA_PATH}/installation/scripts/temp.XXXXXX)
+
+echo "---" >> ${TMPFILE}
+kubectl create cm versions --from-env-file ${VERSIONS_ENV_PATH} -n kyma-installer --dry-run -o yaml >> ${TMPFILE}
+echo "  labels:" >> ${TMPFILE}
```

Comment:
This relies on "metadata" section being last in the file. This seems to be ture currently, but is not granted forever. Please find a way to inject it into "metadata" section reliably.

Reply:
for now i would assume that metadata section is the last one and address it in separate issue